src/components/Slider.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+export default class SimpleSlider extends Component {
+    render() {
+        const settings = {
+            dots: true,
+            infinite: true,
+            speed: 500,
+            slidesToShow: 1,
+            slidesToScroll: 1
+        };
+        return (
+            <div>
+                <Slider {...settings}>
+                    <div>
+                        <img src="https://opencart.templatemela.com/OPC10/OPC100247/OPC1/image/cache/catalog/Main-Banner-2-1800x560.jpg" />
+                    </div>
+                    <div>
+                        <img src="https://opencart.templatemela.com/OPC10/OPC100247/OPC1/image/cache/catalog/Main-Banner-1-1800x560.jpg" />
+                    </div>
+                </Slider>
+            </div>
+        );
+    }
```

Comment:
Can you convert this into a function component? 

Remember when we discussed the class components quickly? its now legacy and we can replace it with functional components instead here is an example

```suggestion

const defaultSettings = {
           dots: true,
            infinite: true,
            speed: 500,
            slidesToShow: 1,
            slidesToScroll: 1
}

interface SliderProps extends React.ReactComponent {
   // you may find SliderSettings types in the library itself. you can check
    settings: SliderSettings
};

export default function SimpleSlider (props:  = defaultSettings) {
        return (
            <div>
                <Slider {...settings}>
                    <div>
                      /* we need this component to be just a slider that we can re-use it anywhere in the application so we will send the children (slides) from top instead of doing this here */
                      {props.children}  
                    </div>
                </Slider>
            </div>
        );
    }
```

Reply:
Oh got u gonna update the code